frontend/src/app/modules/exclusion-dates/custom-date-picker/custom-date-picker.component.ts

```diff
@@ -0,0 +1,54 @@
+import {
+    AfterViewInit,
+    Component, ElementRef,
+    EventEmitter,
+    Input,
+    Output,
+    ViewEncapsulation,
+} from '@angular/core';
+import { DateAdapter, MAT_DATE_FORMATS, MAT_DATE_LOCALE, MAT_NATIVE_DATE_FORMATS } from '@angular/material/core';
+import { MatCalendarCellClassFunction } from '@angular/material/datepicker';
+import { CustomDateAdapter } from '@modules/exclusion-dates/custom-date-adapter';
+import { HeaderDatePickerComponent } from '@modules/exclusion-dates/header-date-picker/header-date-picker.component';
+
+@Component({
+    selector: 'app-custom-date-picker',
+    templateUrl: './custom-date-picker.component.html',
+    styleUrls: ['./custom-date-picker.component.sass'],
+    encapsulation: ViewEncapsulation.None,
+    providers: [
+        { provide: DateAdapter, useClass: CustomDateAdapter, deps: [MAT_DATE_LOCALE] },
+        { provide: MAT_DATE_FORMATS, useValue: MAT_NATIVE_DATE_FORMATS },
+    ],
+})
+export class CustomDatePickerComponent implements AfterViewInit {
+    todayDate: Date;
+
+    @Input() selected: Date | null;
+
+    @Output() selectedChange = new EventEmitter<Date | null>();
+
+    customHeader = HeaderDatePickerComponent;
+
+    constructor(private element: ElementRef) {
+        this.todayDate = new Date();
+        this.todayDate.setHours(0, 0, 0, 0);
+    }
+
+    ngAfterViewInit(): void {
+        const todayDayLine = document.createElement('div');
+
+        todayDayLine.classList.add('today-line');
+        this.element.nativeElement.getElementsByClassName('mat-calendar-body-today').item(0)?.appendChild(todayDayLine);
```

Comment:
In order to add small line below the current day
![image](https://user-images.githubusercontent.com/46250167/188968674-14de6aec-26a1-4427-9608-a3b0ccbc9ddb.png)
I couldn't use ::after or ::before because because these pseudo classes were already used by matdatepicker

Reply:
I have found much better solution. I used text-decoration and text-underline-offset css properties. Fixed it)